ApiFirst/AppModels/AppModels/Orders/CreateOrder.cs

```diff
@@ -8,16 +8,14 @@ public class CreateOrder
         public CreateOrder(int quotes, decimal unitPrice, DateTime liquidateAt, OrderDirection direction, long productId, long portfolioId)
         {
             Quotes = quotes;
-            UnitPrice = unitPrice;
-            NetValue = quotes * unitPrice;
+            NetValue = unitPrice * quotes;
             LiquidateAt = liquidateAt;
             Direction = direction;
             ProductId = productId;
             PortfolioId = portfolioId;
         }
 
         public int Quotes { get; set; }
-        public decimal UnitPrice { get; set; }
         public decimal NetValue { get; set; }
         public DateTime LiquidateAt { get; set; }
```

Comment:
```suggestion
        public DateTime LiquidatedAt { get; set; }
```
Corrigir nas outras entidades e no banco de dados também